electron-app/magnifier/rust-sampler-manager.test.ts

```
@@ -0,0 +1,519 @@
+        this.emit('exit', null, this.signalCode);
+        this.emit('close', null, this.signalCode);
+      }, 10);
+    }
+    return true;
+  }
+}
+
+describe('RustSamplerManager', () => {
+  let manager: RustSamplerManager;
+  let mockProcess: MockChildProcess;
+  let spawnMock: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)